### PR TITLE
bind score docs

### PR DESCRIPTION
**Pull request by camilajenny**
image

### CODE REVIEW

**Review by mkukielka — COMMENTED** [review]
I think, we should adapt to current notation of documentation. So name_of_param: type_of_param ( default='sth' if exists )

e.g.
```
"""Returns pairs of atoms which are within close contac distance cutoff.
    The cutoff is semi-inclusive, i.e (cutoff_low, cutoff].
    Parameters
    ----------
    x, y : atom_dict-type numpy array
        Atom dictionaries generated by oddt.toolkit.Molecule objects.
    cutoff : float
        Cutoff distance for close contacts
    x_column, ycolumn : string, (default='coords')
        Column containing coordinates of atoms (or pseudo-atoms,
        i.e. ring centroids)
    cutoff_low : float (default=0.)
        Lower bound of contacts to find (exclusive). Zero by default.
        .. versionadded:: 0.6
    Returns
    -------
    x_, y_ : atom_dict-type numpy array
        Aligned pairs of atoms in close contact for further processing.
    """
```

**Comment by camilajenny** [comment]
@mkukielka done

File: oddt/docking/BindScore.py [hunk 1]
```diff
@@ -26,6 +43,17 @@ def __init__(self, rec, lig, weights=None):
...
+        """
+
+        Parameters
+        ----------
+        lig: oddt.toolkits.{rdkit,ob}.Molecule
+            ligand molecule used for docking
+
+        Returns
+        -------
+        num_rotors: number
```
Comment (mkukielka) [inline oddt/docking/BindScore.py]: more precise, int or float?

File: oddt/docking/BindScore.py [hunk 2]
```diff
@@ -51,6 +79,21 @@ def _get_close_neighbors(self, molecule, a_idx, num_bonds=1):
         return atoms
 
     def correct_radius(self, atom_dict):
+        """
+        correct radii of atoms to prepare them for scoring. The elements of concern are all metals and some nonmetals,
```
Comment (mkukielka) [inline oddt/docking/BindScore.py:87]: radii?

File: oddt/docking/BindScore.py [hunk 3]
```diff
@@ -69,22 +112,75 @@ def correct_radius(self, atom_dict):
         return atom_dict
 
     def score(self):
+        """
+        score of a bind between molecules
+
+        Returns
+        -------
+        number
```
Comment (mkukielka) [inline oddt/docking/BindScore.py]: again

File: oddt/docking/BindScore.py [hunk 4]
```diff
@@ -4,6 +4,23 @@
...
+        rec: oddt.toolkits.{rdkit,ob}.Molecule
+            receiver molecule (can be a protein) used for docking
+        lig: oddt.toolkits.{rdkit,ob}.Molecule
+            ligand molecule used for docking
+        weights: numpy.array (default=[-0.0356, -0.00516, 0.840, -0.0351, -0.587, 0.0585])
+            weights for energy components
+
+        Returns
+        -------
+        BindScore
```
Comment (mkukielka) [inline oddt/docking/BindScore.py]: param: type
    description

**Review by mkukielka — CHANGES_REQUESTED** [review]
I know, I am a little bit harsh, but It would be nice to provide nicely done docs.

File: oddt/docking/BindScore.py (hunk 2 above)
Comment (camilajenny) [reply oddt/docking/BindScore.py:87]: plural of radius

**Comment by mkukielka** [comment]
conflicts, rebase to master, requested changes?

**Comment by mkukielka** [comment]
If you look into current code of CustomEngine I don't know whether we still need this class.

**Comment by camilajenny** [comment]
> If you look into current code of CustomEngine I don't know whether we still need this class.

Oh, I just rebased it anyway

**Comment by camilajenny** [comment]
Closing as not needed

**Pull request closed without merging**